src/job/job.spec.js

```diff
@@ -0,0 +1,62 @@
+/* eslint-env jest */
+
+import { omit } from "lodash";
+
+import config from "../_config";
+import { xo, testWithOtherConnection } from "../util";
+
+const adminUser = {
+  email: "admin2@admin.net",
+  password: "admin",
+  permission: "admin",
+};
+
+const defaultJob = {
+  job: {
+    name: "jobTest",
+    timeout: 2000,
+    type: "call",
+    key: "snapshot",
+    method: "vm.snapshot",
+  },
+};
+
+describe("job", () => {
+  describe(".create() :", () => {
+    it("creates a new job", async () => {
+      defaultJob.job.paramsVector = {
+        type: "cross product",
+        items: [
+          {
+            type: "set",
+            values: [
+              {
+                id: config.vmIdXoTest,
+                name: "snapshot",
+              },
+            ],
+          },
+        ],
+      };
+      const jobId = await xo.createJob(defaultJob);
+      expect(typeof jobId).toBe("string");
+      expect(
+        omit(await xo.call("job.get", { id: jobId }), "id", "userId")
+      ).toMatchSnapshot();
+    });
+
+    it("creates a new job and tests the userId", async () => {
+      const userId = await xo.createUser(adminUser);
+      const { email, password } = adminUser;
+      await testWithOtherConnection({ email, password }, async xo => {
+        const jobId = await xo.createJob(defaultJob);
+        const job = await xo.call("job.get", { id: jobId });
+        expect(job.userId).toBe(userId);
+      });
+    });
+
```

Comment:
Please add a test which test a job containing a user id.

Reply:
A failed test with a non admin user ?

Reply:
I think it's already done...